ethpm_types/contract_type.py

```diff
@@ -128,18 +128,39 @@ class ContractInstance(BaseModel):
 
 
 class SourceMapItem(BaseModel):
+    """
+    An object modeling a node in a source map; useful for mapping
+    the source map string back to source code.
+    """
+
     # NOTE: `None` entry means this path was inserted by the compiler during codegen
     start: Optional[int]
+    """
+    The byte-offset start of the range in the source file.
+    """
+
     stop: Optional[int]
+    """
+    The byte-offset length.
```

Comment:
Hmm, maybe we need to re-name this field

Reply:
yes I was going to ask about that!

Reply:
Renamed to `length` because it matches other similar field names in EthPMTypes, such as the one in `LinkDependencies`. It also aligns with the documentation of the feature (https://docs.soliditylang.org/en/v0.8.18/internals/source_mappings.html) and makes it easier to follow what is going on. `size` is a better name in my opinion if these other things weren't true.

Reply:
also this was a breaking change I suppose, so I put the `!`